.github/workflows/e2e.yaml

```diff
@@ -75,3 +75,166 @@ jobs:
+        apiVersion: kuma.io/v1alpha1
+        kind: ContainerPatch
+        metadata:
+          name: disable-init-container
+          namespace: kuma-system
+        spec:
+          initPatch:
+          - op: replace
+            path: /command
+            value: '["echo"]'
+        EOF
+        
+        kubectl apply -f /tmp/container-patch.yaml
+        rm -f /tmp/container-patch.yaml
+    - name: annotate default namespace to automatically inject kuma to pods it contains
+      run: |
+        kubectl label ns default kuma.io/sidecar-injection=enabled
+    - name: deploy test apps
+      run: |
+        # server
+        
+        echo '---
+        apiVersion: v1
+        kind: Service
+        metadata:
+          name: example-server
+        spec:
+          ports:
+          - port: 80
+            name: http
+            appProtocol: http
+          selector:
+            service: example-server
+        ---
+        apiVersion: apps/v1
+        kind: Deployment
+        metadata:
+          name: example-server
+          labels:
+            service: example-server
+        spec:
+          selector:
+            matchLabels:
+              service: example-server
+          template:
+            metadata:
+              labels:
+                service: example-server
+            spec:
+              terminationGracePeriodSeconds: 0
+              containers:
+              - name: nginx
+                image: "nginx:stable"
+                imagePullPolicy: IfNotPresent
+                ports:
+                - containerPort: 80
+                readinessProbe:
+                  httpGet:
+                    path: /
+                    port: 80
+                  initialDelaySeconds: 3
+                  periodSeconds: 3
+                livenessProbe:
+                  httpGet:
+                    path: /
+                    port: 80' | kubectl apply -f -
+        
+        # client
+        
+        echo '---
+        apiVersion: apps/v1
+        kind: Deployment
+        metadata:
+          name: example-client
+          labels:
+            service: example-client
+        spec:
+          strategy:
+            rollingUpdate:
+              maxSurge: 1
+              maxUnavailable: 0
+          selector:
+            matchLabels:
+              service: example-client
+          template:
+            metadata:
+              labels:
+                service: example-client
+            spec:
+              terminationGracePeriodSeconds: 0
+              containers:
+              - name: curl
+                image: curlimages/curl
+                command: ["/bin/sleep", "infinity"]
+                imagePullPolicy: IfNotPresent' | kubectl apply -f -
+        
+        # wait till our example deployment are fully ready
+        kubectl rollout status deployment example-server
+        kubectl rollout status deployment example-client
+    - name: test connect without Merbridge
+      run: |
+        sleep 5
```

Comment:
It is recommended not to use sleep as a condition for Pod ready, it is better to judge whether the Pod status is running.

Reply:
Ack. I'm gonna fix it

Reply:
e2e test still failed.  😂

If sleep is ok, we can keep this.

Reply:
ok, finally done!